From 87eda7b9c9eaa1bfa20224a6b4799c665fafe2ae Mon Sep 17 00:00:00 2001
From: Gang G Chen <gang.g.chen@intel.com>
Date: Mon, 25 Mar 2019 08:28:01 +0000
Subject: [PATCH] Change keymaster configuration for keymaster test

Signed-off-by: Gang G Chen <gang.g.chen@intel.com>
---
 android_keymaster/keymaster_configuration.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/android_keymaster/keymaster_configuration.cpp b/android_keymaster/keymaster_configuration.cpp
index 35b83cc..0d511e1 100644
--- a/android_keymaster/keymaster_configuration.cpp
+++ b/android_keymaster/keymaster_configuration.cpp
@@ -22,7 +22,9 @@
 #include <regex.h>
 
 #define LOG_TAG "keymaster"
-#include <log/log.h>
+
+#define ALOGE printf
+#define ALOGI printf
 
 #ifndef KEYMASTER_UNIT_TEST_BUILD
 #include <cutils/properties.h>
-- 
2.20.1

